### PR TITLE
feat: add more sort order values

### PR DESCRIPTION
**Pull request by barnabasJ**

### CODE REVIEW

**Comment by zachdaniel** [comment]
I like this, but I think we might want to expose it as `NULL` instead of `NIL` in the graphql layer.

File: lib/api/api.ex [hunk 1]
```diff
@@ -200,15 +200,15 @@ defmodule AshGraphql.Api do
           module: schema,
           identifier: :desc_nils_first,
           __reference__: AshGraphql.Resource.ref(env),
-          name: "DESC_NILS_FIRST",
-          value: :desc
+          name: "DESC_NULLS_FIRST",
+          value: :dialyzer_succ_typings
```
Comment (zachdaniel) [inline lib/api/api.ex]: This looks like an accident 😆

Comment (barnabasJ) [reply lib/api/api.ex]: I blame copilot :rofl:

**Comment by zachdaniel** [comment]
🚀 Thank you for your contribution! 🚀

**Pull request merged**